src/Log/LogService.php

```diff
@@ -164,6 +164,8 @@ public function createLog($name, $priority = null, $format = null, $filename = n
 
         if ($format === null) {
             $format = $logConfig->format;
+        } else {
```

Comment:
I suggest deleting this change as well. The format is used in one place only, inside `createLogger`. Why not calling the function replacing %runId% there, right before it is being used? That way we don't have to mix code getting the format and preparing the format. Keeping it simple with less special treatment code.

Reply:
Yes, a nice suggestion. But some tests will have to be changed accordingly.